datasets/glue/README.md

```diff
@@ -39,9 +103,55 @@
 
 ### Dataset Summary
 
-GLUE, the General Language Understanding Evaluation benchmark
-(https://gluebenchmark.com/) is a collection of resources for training,
-evaluating, and analyzing natural language understanding systems.
+GLUE, the General Language Understanding Evaluation benchmark (https://gluebenchmark.com/) is a collection of resources for training, evaluating, and analyzing natural language understanding systems.
+
+#### ax
+
+A manually-curated evaluation dataset for fine-grained analysis of system performance on a broad range of linguistic phenomena. This dataset evaluates sentence understanding through Natural Language Inference (NLI) problems. Use a model trained on MulitNLI to produce predictions for this dataset.
+
+#### cola
+
+The Corpus of Linguistic Acceptability consists of English acceptability judgments drawn from books and journal articles on linguistic theory. Each example is a sequence of words annotated with whether it is a grammatical English sentence.
+
+#### mnli
+
+The Multi-Genre Natural Language Inference Corpus is a crowdsourced collection of sentence pairs with textual entailment annotations. Given a premise sentence and a hypothesis sentence, the task is to predict whether the premise entails the hypothesis (entailment), contradicts the hypothesis (contradiction), or neither (neutral). The premise sentences are gathered from ten different sources, including transcribed speech, fiction, and government reports. We use the standard test set, for which we obtained private labels from the authors, and evaluate on both the matched (in-domain) and mismatched (cross-domain) section. We also use and recommend the SNLI corpus as 550k examples of auxiliary training data.
```

Comment:
The "we" here sounds a bit weird as it could make the reader thing it's we hugging face when it's we the authors of the dataset instead.